CDP4CommonView/Diagram/Views/Cdp4DiagramConnector.cs

```diff
@@ -193,7 +193,7 @@ private void Initialize()
             BindingOperations.SetBinding(this, TargetProperty, targetBinding);
 
             var textBinding = ViewUtils.CreateBinding(this.DataContext, "DisplayedText", BindingMode.OneWay);
-            BindingOperations.SetBinding(this, TextProperty, textBinding);
+            BindingOperations.SetBinding(this, ContentProperty, textBinding);
```

Comment:
Can you verify please that this did not break anything too much in the diagramming?